projects/core/src/lib/utils/dom/hotkey.ts

```diff
@@ -0,0 +1,44 @@
+export const enum HotkeyModifier {
+    CTRL = 0b0001,
+    ALT = 0b0010,
+    SHIFT = 0b0100,
+    META = 0b1000,
```

Comment:
Cool idea to use bit **mask** inside textfield **mask** 😄 👍 

**nitpicking:**

```suggestion
    CTRL = 1 << 0,
    ALT = 1 << 1,
    SHIFT = 1 << 2,
    META = 1 << 3,
```

It is just the different way to do the same thing but it is easier to extend enum with new values.